tests/e2e/framework/framework.go

```diff
@@ -185,10 +185,7 @@ func (t *testCleanup) cleanup() {
 // Fetch and save cluster pod logs using kuebctl
 // Logs are uploaded during test tear down
 func (c *CommonConfig) saveLogs(r int) error {
-	if c.Cleanup.skipCleanup {
```

Comment:
Please add another way to skip that rather than delete the feature entirely - it makes the dev mode (where you repeatedly run 1 test) a lot slower if there is no way to skip that stage

Reply:
Log fetching and cleanup skipping are very different things. I did try running the tests with this - I don't see any significant difference, but if someone wants to add a 'skipLogs' flag I don't mind (but please let the default to on - it is important to get the logs so we can see any unexpected errors ).

Reply:
until recently none of the e2e ran with skip_cleanup which was a dev only shortcut so it'd be nice to preserve the dev feature (short dev+test cycles) and add some new stuff for ci as needed (and yes I agree it could be 2 flags instead of 1, but as long as we don't lose a way to get into that state)